src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java

```diff
@@ -136,6 +138,16 @@ public void onGuiChat(ClientChatReceivedEvent e) {
 		String unformatted = Utils.cleanColour(e.message.getUnformattedText());
 		Matcher matcher = SLAYER_XP.matcher(unformatted);
 		if (unformatted.startsWith("You are playing on profile: ")) {
+			if(NotEnoughUpdates.INSTANCE.config.notifications.doBoosterNotif) {
```

Comment:
im pretty sure you dont get a "You are playing on profile" chat message if you don't have multiple profiles, so you might want to use an WorldEvent.Load or an EntityJoinWorldEvent with an extra check for `if(event.entity == Minecraft.getMinecraft().thePlayer)` or even a new tick event handler in your CookieWarning.java that checks every 1200 ticks.

Reply:
I changed it to use EntityJoinWorldEvent